next.config.js

```diff
@@ -2,7 +2,10 @@
 module.exports = {
   reactStrictMode: true,
   images: {
-    domains: ["book-bazar-images.s3.us-east-2.amazonaws.com"],
+    domains: [
+      'book-bazar-images.s3.us-east-2.amazonaws.com',
+      'campusstore.mcmaster.ca',
+    ],
```

Comment:
Will need to add the Google books image domain if we ever fall back to that. (Maybe we should prioritize it over the campus store one too? Idk)